packages/server/src/restaurants/domain/GetRestaurantHandler.ts

```diff
@@ -15,7 +15,7 @@ class GetRestaurantHandler implements Handler<GetRestaurantRequest, Restaurant |
 
   async exec({ id, profileMustBeCompleted = true }: GetRestaurantRequest): Promise<Restaurant | null> {
     const restaurant = await this.restaurantModel.findById(id);
-    if (!restaurant || (profileMustBeCompleted && !restaurant?.profileCompleted)) return null;
+    if (!restaurant || (profileMustBeCompleted && !restaurant?.verified)) return null;
```

Comment:
Tu nie będzie `isCompleted`? Pytam bo nie jestem na bieżąco z ustaleniami. A jak ma być `verified`, to może lepiej nazwę argumentu zrobić `profileMustBeCompleted` -> `profileMustBeVerified`.